src/queries.js

```diff
@@ -173,6 +171,7 @@ export {
   queryByPlaceholderText,
   getByPlaceholderText,
   queryByText,
+  getText,
```

Comment:
This file should _only_ expose queries. Other utilities should live in other files. Could you move `getText` to its own file?

Reply:
Got it. Also, I wonder if we should publicly call it `getNodeText` instead?

Reply:
Agreed. We can rename it to `getNodeText` :+1: